tests/unit/command_test.ts

```diff
@@ -705,6 +705,7 @@ Deno.test(
 Deno.test(
   { permissions: { run: true, read: true, env: true } },
   async function commandClearEnv() {
+    Deno.env.set("DENO_COMMAND_CLEAR_ENV_TESTING", "TESTING");
```

Comment:
These should be done in `try/finally` block otherwise they will start leaking. Why are these needed anyway? Can't they be set in `env` option of the spawned subprocess?